src/ToolInfo.php

```diff
@@ -90,7 +90,7 @@ public function isInstalledAsPhar()
     public function isInstalledByComposer()
     {
         if (null === $this->isInstalledByComposer) {
-            $this->isInstalledByComposer = !$this->isInstalledAsPhar() && file_exists($this->getComposerInstalledFile());
+            $this->isInstalledByComposer = !$this->isInstalledAsPhar() && @file_exists($this->getComposerInstalledFile());
```

Comment:
supress `open_basedir` warning

Reply:
What warning? I'd expect some test added to cover this change.

Reply:
File access outside `open_basedir` :) I won't write test for this as it will require starting another process.

Reply:
So if in a few months someone will remove that `@` all will be fine 😕

Reply:
Is there any test which tests console with a new php process for each test?

Reply:
why `@` ? isn't warning collected by our `set_error_handler` hook>
no tests, no merge

no, you need to write one from scratch

Reply:
The issue is that this only happens if you config `open_basedir` through the PHP `ini` system, which is a pain to do in unit tests. When trying to detect something like we do here what is the harm suppressing the warning?

To reproduce, you configure `open_basedir` to some directory outside of the project install for your CLI and run the tool.

Reply:
@keradus do you know a way to test with different ini values for PHP within a single PHPUnit session?

Reply:
i was not checking for that possibility myself